[WIP] Additional baseline features

john-westcott-iv opened the pull request:
This work is moving additional code into DAB to support a standard service "template" for AAP.
The goal of the templated app is to give a developer standards such as:
* Standard models: User, Team, Organization
* Our filtering/paging/etc turned on
* Our API templates
* Integration with RBAC/Resource Sync

This is still a work in progress and very TBD.

john-westcott-iv committed:
Adding additional baseline features for all applications

john-westcott-iv committed:
Make the api_*root_view non-changeable

john-westcott-iv committed:
Adding standard views

john-westcott-iv committed:
Fixing default value of ANSIBLE_BASE_BASIC_AUTH_ENABLED in second location

john-westcott-iv committed:
Adding defaut session handler

john-westcott-iv committed:
Make the rest template logout return you to the same location in the API

relrod commented:
@john-westcott-iv can I ask for a more descriptive PR title and description here? :smile:

john-westcott-iv committed:
Create django_template app

bhavenst committed:
Add common JWT auth with permissions enabled

bhavenst committed:
(TODO) remove social auth import

This is included to make things work for now, but probably not the
proper way to go.

john-westcott-iv commented:
Closing this issue as we've not really been using the template for services.
